### PR TITLE
Added jekyll-sitemap to generate sitemap.xml. relates to #16

### PR DESCRIPTION
**Pull request by Ryuno-Ki**
Don't forget to run `bundle`!

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -47,11 +49,12 @@ PLATFORMS
 DEPENDENCIES
   jekyll (= 3.4.3)
   jekyll-feed (~> 0.6)
+  jekyll-sitemap (~> 1.1)
   minima (~> 2.0)
   tzinfo-data
 
 RUBY VERSION
-   ruby 2.4.0p0
+   ruby 2.1.9p490
```
Comment (Ryuno-Ki) [inline Gemfile.lock]: I'm not sure, whether this could cause troubles …

File: Gemfile.lock [hunk 2]
```diff
@@ -47,11 +49,12 @@ PLATFORMS
...
 
 BUNDLED WITH
-   1.14.2
+   1.15.1
```
Comment (Ryuno-Ki) [inline Gemfile.lock]: I'm not sure, whether this could cause troubles …

**Comment by Ryuno-Ki** [comment]
Currently, the Gem seem not to support subdomains, because it can't „see” them. I have to research how to deal with that. Maybe generating dedicated sitemap.xml for docs.halium.org + referencing it in #18 could be sufficient.

File: Gemfile.lock (hunk 1 above)
Comment (bhush9) [inline Gemfile.lock]: Can you try to stash those changes away? and just keep the jekyll* related changes and see if it still works?

**Comment by bhush9** [comment]
um wat. github is funny

**Pull request merged**